src/server.c

```diff
@@ -585,6 +606,18 @@ dictType keylistDictType = {
     NULL                        /* allow to expand */
 };
 
+/* KeyDict hash table type has unencoded redis objects as keys and
+ * dicts as values. It's used for PUBSUB command to track clients subscribing the channels. */
+dictType keydictDictType = {
```

Comment:
KeyDict seems to be a weird name to me. 

Maybe `objToDictDictType` or `channelToClientsDictType`, WDYT?